packages/react/src/PointerBox/PointerBox.test.tsx

```diff
@@ -1,56 +1,61 @@
-import {PointerBox} from '..'
-import {render, behavesAsComponent, checkExports, renderStyles} from '../utils/testing'
-import {render as HTMLRender} from '@testing-library/react'
-import axe from 'axe-core'
+import {describe, expect, it} from 'vitest'
+import {render} from '@testing-library/react'
+import {PointerBox, ThemeProvider} from '..'
+import theme from '../theme'
 
 describe('PointerBox', () => {
-  behavesAsComponent({Component: PointerBox})
-
-  checkExports('PointerBox', {
-    default: PointerBox,
-  })
-
   it('renders a <Caret> in <Box> with relative positioning', () => {
-    expect(render(<PointerBox />)).toMatchSnapshot()
-  })
-
-  it('should have no axe violations', async () => {
-    const {container} = HTMLRender(<PointerBox />)
-    const results = await axe.run(container)
-    expect(results).toHaveNoViolations()
+    expect(render(<PointerBox theme={theme} />)).toMatchSnapshot()
   })
 
   it('applies the border color via "borderColor" prop for backwards compatibility', () => {
-    expect(render(<PointerBox borderColor="danger.emphasis" />)).toMatchSnapshot()
+    expect(render(<PointerBox borderColor="danger.emphasis" theme={theme} />)).toMatchSnapshot()
   })
 
   it('applies the border color via sx prop', () => {
-    expect(render(<PointerBox sx={{borderColor: 'danger.emphasis'}} />)).toMatchSnapshot()
+    expect(render(<PointerBox sx={{borderColor: 'danger.emphasis'}} theme={theme} />)).toMatchSnapshot()
   })
 
   it('applies the background color via "bg" prop for backwards compatibility', () => {
-    expect(render(<PointerBox bg="danger.emphasis" />)).toMatchSnapshot()
+    expect(
+      render(
+        <ThemeProvider theme={theme}>
+          <PointerBox bg="danger.emphasis" />
+        </ThemeProvider>,
+      ),
+    ).toMatchSnapshot()
```

Comment:
Update this test from a snapshot test to one that is checking the background color